subsys/nfc/nfc_platform_zephyr.c

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
It should be nfc/platform_zephyr.c (to align with our naming scheme)

Does it make sense to add another directory called ```lib``` or ```stack```? It would be more clear that this source file is necessary if you want to use nrfxlib NFC stack/libs.

Reply:
Ok renamed and moved to lib directory